qatrack/service_log/signals.py

```
@@ -41,3 +43,29 @@ def on_service_event_saved(*args, **kwargs):
+@receiver(post_delete, sender=models.ReturnToServiceQA, dispatch_uid="sl_update_rts_cache_post_delete")
+def update_rts_cache(sender, **kwargs):
+    """When a RTS is completed invalidate the unreviewed counts"""
+    cache.delete(settings.CACHE_RTS_QA_COUNT)
+    cache.delete(settings.CACHE_RTS_INCOMPLETE_QA_COUNT)
+    cache.delete(settings.CACHE_SL_NOTIFICATION_TOTAL)
+
+
+@receiver(post_save, sender=models.ServiceEventStatus, dispatch_uid="sl_update_se_cache_ses_post_save")
+@receiver(post_delete, sender=models.ServiceEventStatus, dispatch_uid="sl_update_se_cache_ses_post_delete")
+@receiver(post_delete, sender=models.ServiceEvent, dispatch_uid="sl_update_se_cache_se_post_delete")
+def update_se_cache(sender, **kwargs):
```

Ruff (0.15.18)

[warning] Unused function argument: `sender`

(ARG001)